views/partials/header.ejs

```diff
@@ -0,0 +1,10 @@
+<h1>The Jobs EJS Application</h1>
+<% if (errors) {
+  errors.forEach((err) => { %>
+    <div>Error: <%= err %></div>
+<% }) } %>
+<% if (info) {
+  info.forEach((msg) => { %>
+    <div>Info: <%= msg %></div>
+<% }) } %>
```

Comment:
Requires change:
These if conditions are always true because `errors` and `info` are initially empty arrays and empty arrays are defined and therefore `true`. Instead, you'll want to check if defined AND has length:
```ejs
<% if (errors && errors.length) {
    errors.forEach((err) => { %>
      <p class="alert alert-danger">Error: <%= err %></p>
<% }) } %>
<% if (info && info.length) {
    info.forEach((msg) => { %>
      <p class="alert alert-success">Info: <%= msg %></p>
<% }) } %>
```
🗒️ You're currently importing Bootstrap CSS within the `head.ejs` file, so utilize it! For example, leverage Bootstrap's [alerts](https://getbootstrap.com/docs/5.3/components/alerts/#examples).